html/lankar.html

```diff
@@ -36,10 +36,50 @@
     <main>
 
         <ul class="breadcrumbs">
-            <li>Startsida / </li>
-            <li></li>
+            <li><a href="/index.html">Startsida</a> / </li>
+            <li>Länkar</li>
         </ul>
 
+        <h1>Länkar</h1>
+
+            <h2>Länkar till andra myndigheter och samarbetspartners</h2>
+                <ul>
+                    <li><a href="https://www.regeringen.se/sveriges-regering/kulturdepartementet">Kulturdepartementet</a></li>
+                    <li><a href="https://www.kulturradet.se/">Statens Kulturrådet</a></li>
+                    <li><a href="https://statenskonstrad.se/">Statens Konstråd</a></li>
+                    <li><a href="https://musikverket.se/">Statens Musikverk</a></li>
+                    <li><a href="https://kulturanalys.se/">Myndigheten för Kunturanalys</a></li>
+                    <li><a href="https://www.filminstitutet.se/">Svenska Filminstitutet</a></li>
+                    <li><a href="https://www.raa.se/">Riksantikvarieämbetet</a></li>
+                    <li><a href="https://riksarkivet.se/">Riksarkivet</a></li>
+                    <li><a href="https://tillvaxtverket.se/">Tillväxtverket</a></li>
+                    <li><a href="https://www.svff.se/">Sveriges Författarfond</a></li>
+                    <li><a href="https://www.vr.se/">Vetenskapsrådet</a></li>
+                </ul>
+
+
+
+            <h2>Internationellt utbyte och samarbete</h2>
+                <p>Konstnärsnämnden arbetar för att öka internationella oh interkulturella samarbeten inom kulturlivet. Det sker genom bidragsgivning, samordning och samarbeten med olika institutioner och organisationer.</p>
+
+            <h2>Internationella länkar</h2>
+
+                <ul>
+                    <li><a href="https://www.kulturradet.se/kreativa-europa/">EU - Kulturkontakt Sverige</a></li>
+                    <li><a href="https://ec.europa.eu/culture/">Europeiska Komissionen Kultur</a></li>
+                    <li><a href="https://icom.museum/en/">ICOM</a></li>
+                    <li><a href="https://www.icorn.org/">ICORN</a></li>
+                    <li><a href="https://ifacca.org/">IFACCA</a></li>
+                    <li><a href ="https://www.ifla.org/">IFLA</a></li>
+                    <li><a href="http://www.kulturkontaktnord.org/">Kulturkontakt Nord</a></li>
```

Comment:
This link is not working. I think they changed it to:
[https://www.nordiskkulturkontakt.org/sv/](url)

Reply:
Until wednesday it was all working. For me it doesn't open yours or mine. Should we take away this link?

Reply:
The link I wrote here is working for me. I'll change it directly here on github and test it. If it works I'll merge this branch, otherwise Ill just remove the entire link. Everything else is looking great :)